src/sqlfluff/dialects/bigquery_keywords.py

```diff
@@ -0,0 +1,163 @@
+"""A list of all BigQuery SQL key words."""
+
+bigquery_reserved_keywords = """ALL
```

Comment:
Is there a source for these words, like a BigQuery documentation page? If so, it'd be cool to link to that to simplify potential future updates to this list.

Reply:
Good idea - added!